hack/Makefile

```diff
@@ -0,0 +1,11 @@
+tests = $(shell ./run_test --list 2>/dev/null)
+
+.PHONY: all
+
+all: $(tests)
+
+%:
+	@if [[ "$(tests)" =~ "$@" ]]; then \
+		echo Running test: $@; \
+		VERSION=$(VERSION) IMAGE_NAME=$(IMAGE_NAME) ./run_test $@; \
+	fi;
```

Comment:
>can not this be directly in common.mk?
Will try to incorporate it.

>What is the check '=~' for? It guards against misspelled test names?
Yes, although changing it to multiple targets for the rule makes more sense so I will do that.